src/commands/mod.rs

```diff
@@ -48,30 +49,22 @@ impl Command {
                 annotations,
                 reference,
                 host_id,
-                link_name,
+                provider_id,
                 ..
             }) => {
                 let mut annotations = annotations.to_owned();
                 insert_managed_annotations(&mut annotations, model_name);
                 Some((
                     Event::ProviderStarted(ProviderStarted {
+                        provider_id: provider_id.to_owned(),
                         annotations: annotations.to_owned(),
-                        claims: ProviderClaims::default(),
+                        claims: None,
```

Comment:
It will not, since the `evt_matches_expected` ignores the claims. We could probably get rid of this soon, we just kept it for providers since those are notoriously larger and we don't want the scalers to keep requesting to start providers when getting heartbeats